src/compiler/checker.ts

```diff
@@ -29277,6 +29285,28 @@ namespace ts {
             }
         }
 
+        function superCallIsRootLevelInConstructor(superCall: Node, body: Block) {
+            let expressionParent = superCall.parent;
+
+            while (isParenthesizedExpression(expressionParent)) {

```

Comment:
We already have a function that does this, called `walkUpParenthesizedExpressions`.